test/testing_suite_test.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:test/test.dart';
+
+import 'utils/client.dart'; // testServer
+import "tests/analytics.dart"; // versionTests(), analyticsTests()
+import "tests/documents.dart"; // documentsTests()
+import "tests/dump.dart"; // dumpTests()
+import "tests/exceptions.dart"; // exceptionsTests()
```

Comment:
I found the "test/tests" path a little redundant but I have no idea what should we choose instead :/